pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey @dheerajodha, please have a look at the following snippet. It should help you add the required configuration for tidy pom :) 

```xml
...
<build>
		<plugins>
			<plugin>
				<groupId>org.codehaus.mojo</groupId>
				<artifactId>tidy-maven-plugin</artifactId>
				<version>1.1.0</version>
				<executions>
					<execution>
						<id>tidy-pom</id>
						<phase>validate</phase>
						<goals>
							<goal>pom</goal>
						</goals>
					</execution>
				</executions>
			</plugin>
			<plugin>
				<groupId>org.springframework.boot</groupId>
				<artifactId>spring-boot-maven-plugin</artifactId>
			</plugin>
		</plugins>
	</build>
...
```

Reply:
@alecharp please feel free to correct me if this is not what you had in mind.

Reply:
Thank you @ADI10HERO for your help.
This is very neat, makes sense to me. Now I tried to search these tags on Google and found out that I could have gotten them on the plugin's own page [here](https://www.mojohaus.org/tidy-maven-plugin/usage.html). I'll make sure to check this area next time.

Reply:
No problem and happy to help @dheerajodha

Reply:
I deliberately didn't put the XML to use so that you can find what you can use (part of learning) and that you (Dheeraj) can come with a personal solution to the problem I pointed ;)

Reply:
Thank you very much for being so thoughtful about my learning, I really appreciate this!! :-)
I'll now keep this in mind and try harder next time.